MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3490,6 +3491,13 @@ public string GetVideoTransposeDirection(EncodingJobInfo state)
                 }
 
                 mainFilters.Add(tonemapArgs);
+
+                // convert back to nv12 for VAAPI encoders
+                // Q: Is this still being used?
+                if (requireDoviReshaping && isVaapiEncoder)
+                {
+                    mainFilters.Add("format=" + outFormat);
+                }
```

Comment:
Seems not. `tonemapx` does not exist in vanilla ffmpeg.

https://github.com/jellyfin/jellyfin/blob/ae82a4eee02060e6ff56a23fdf51ed47f17ddb9c/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L4477-L4497

Reply:
Removed for now. If it causes regression we can add it back.